docker/data_source_docker_plugin.go

```diff
@@ -46,22 +46,27 @@ func dataSourceDockerPlugin() *schema.Resource {
 	}
 }
 
-func getDataSourcePluginKey(d *schema.ResourceData) string {
-	if id, ok := d.GetOk("id"); ok {
-		return id.(string)
+var errDataSourceKeyIsMissing = errors.New("One of id or alias must be assigned")
```

Comment:
You should replace this with the [ExactlyOneOf](https://github.com/hashicorp/terraform-plugin-sdk/blob/aa07c4354f9a8fd948ed40f020f340132ada44bc/helper/schema/schema.go#L196) attribute property

Reply:
```
❯ go vet ./...
# github.com/terraform-providers/terraform-provider-docker/docker
docker/data_source_docker_plugin.go:19:5: unknown field 'ExactlyOneOf' in struct literal of type schema.Schema
docker/data_source_docker_plugin.go:24:5: unknown field 'ExactlyOneOf' in struct literal of type schema.Schema
# github.com/terraform-providers/terraform-provider-docker/docker
vet: docker/data_source_docker_plugin.go:19:5: unknown field ExactlyOneOf in struct literal
```

Oh, `ExactlyOneOf` isn't supported.
Probably we have to update the Docker SDK.

Reply:
yes, see https://github.com/kreuzwerker/terraform-provider-docker/pull/102

Reply:
ok let's merge this PR first then #102 and we can the `ExactlyOneOf` property then